lib/cbra/cli.rb

```diff
@@ -1,14 +1,13 @@
 # frozen_string_literal: true
 
 require "thor"
-require "pp"
 
 module Cbra
   # Implements the tool's CLI
   class CLI < Thor
     desc "ls APP_PATH", "Prints the tree of components for an app"
     def ls(app_path)
-      pp ComponentTree.new(app_path).to_h
```

Comment:
* `pp` prints names of components like "This" and "That"
* `puts` prints names of components like This and That

I think the quotes are unnecessary